misc/scripts/deploy-tools.sh

```diff
@@ -210,6 +211,8 @@ function deployESXInstall {
 }
 
 function deployESXPost {
+    # restart hostd so that esxcli extension file is read
+    $SSH $TARGET $HOSTD restart
```

Comment:
Yes. Agree.
Although I am not sure if we can specify hostd restart during VIB installation. 
Currently we do installation as localcli software vib install <vib_name>
Will be good to have it done during installation because it presently it will require manual hostd restart.